zeroex/asset_data_decoder.go

```diff
@@ -55,10 +59,17 @@ type ERC1155AssetData struct {
 type ERC20BridgeAssetData struct {
 	TokenAddress  common.Address
 	BridgeAddress common.Address
-	Calldata      []byte
+	BridgeData    []byte
```

Comment:
ABIs don't hash parameter names into function selectors so this slipped through the cracks. Changing it back to agree with the implementation.